app/knight/knight.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(
+            self,
+            knight_dict: dict

```

Comment:
You should use actual arguments for `Knight` instance initialization, `knight_dict` is not descriptive enough. Besides that, you can use type hinting for separate arguments that you pass to `__init__`.